src/linux/processors.c

```diff
@@ -404,3 +464,66 @@ bool cpuinfo_linux_detect_thread_siblings(
 	}
 }
 
+bool cpuinfo_linux_detect_cluster_cpus(
+	uint32_t max_processors_count,
+	uint32_t processor,
+	cpuinfo_siblings_callback callback,
+	void* context)
+{
```

Comment:
Curly brackets on the same line
```suggestion
	void* context) {
```